src/data/rat/basic.lean

```diff
@@ -661,6 +661,23 @@ begin
   simp [division_def, coe_int_eq_mk, mul_def one_ne_zero d0]
 end
 
+lemma mk_div_mk_cancel_left {n d x : ℤ} (hx : x ≠ 0) : (n /. x) / (d /. x) = n /. d :=
```

Comment:
Do we already have the lemma that says the following?
```lean
lemma mk_mul_mk_cancel {n d x : ℤ} (hx : x ≠ 0) : (n /. x) * (x /. d) = n /. d :=
```

Reply:
I could not find anything, so I added that lemma and refactored the above to be a corollary.
Q: I don't understand the naming convention for `_cancel_left`, `_cancel_right` and just `_cancel`, could you help me with the statement names in the commit below please?